l10n_vn_account_invoice_number/views/account_invoice.xml

```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
The icon could be more "Invoice specific".
How about re-using the icon from odoo/addons/account/static/description/icon.png and set red background and add the star?

Reply:
Nice logo, we could re-use the same trick for many other vn specific modules ;)